include/seqan3/alignment/pairwise/edit_distance_fwd.hpp

```diff
@@ -70,8 +80,6 @@ struct default_edit_distance_trait_type
     using database_type = std::remove_reference_t<database_t>;
```

Comment:
`/\ /\ /\ /\ /\ /\ /\ /\ /\ /\ /\` score_type definition - above this `/\ /\ /\ /\ /\ /\ /\ /\ /\ /\ /\`